analysis/management/commands/multi_character_analysis.py

```diff
@@ -0,0 +1,80 @@
+import os
+from collections import Counter
+from string import ascii_letters, digits, punctuation, whitespace
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models.functions import Length
+from lxml import html
+from nltk.tokenize import stanford_segmenter
+
+from analysis.models import SingleCount, MultiCount
+from dictionary.models import Entry
+
+
+class Command(BaseCommand):
+    help = '''
+    parse a list of html documents and bag the occurence of dictionary entries.  
+    '''
+
+    def __init__(self):
+        self.base_dir = os.path.dirname(os.path.abspath(__file__))
+        os.environ['JAVAHOME'] = settings.JAVAHOME
+        os.environ['CLASSPATH'] = settings.CLASSPATH
+        self.segmenter = stanford_segmenter.StanfordSegmenter(
+            path_to_jar=settings.STFD_SEG,
+            path_to_slf4j=settings.SLF4J,
+            path_to_sihan_corpora_dict=settings.SIHAN_DICT,
+            path_to_model=settings.MODEL,
+            path_to_dict=settings.DICT
+            )
+        self.entries = {e.simple: e for e in Entry.objects.all()}
+        super().__init__()
+
+    def extract_text_from_html(self, file_path=None, encoding='utf-8'):
+        file_path = os.path.join(self.base_dir, 'resources', 'wiki_zh_china.xml') #TODO: Fix this!
+        with open(file_path, encoding="utf-8") as f:
+            article = f.read()
+        tree = html.fromstring(article)
+        content = tree.xpath(r'//*[@id="mw-content-text"]//text()')
+        return ''.join(content)
```

Comment:
I guess for Chinese it doesn't matter, but it might be wiser to insert a space here so the segmenter is aware that the two items are not conjoined. For instance, if you had the string: `<p>Hello there</p><p>Aaron</p>` after running your code the segmenter will see `Hello thereAaron`.